receiver/awscontainerinsightreceiver/internal/gpu/dcgmscraper.go

```diff
@@ -25,7 +26,7 @@ import (
 )
 
 const (
-	caFile                    = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	caFile                    = "/etc/amazon-cloudwatch-observability-agent-cert/tls-ca.crt"
```

Comment:
I guess the changes in this file are coming from https://github.com/aditya-purang/opentelemetry-collector-contrib-TI/commit/9cb314e7d27bfdbcbba3e38f7d4ff86e26f018c4 - can we merge it to `ci-nvidia-gpu` first so they don't show up as diffs in your PR?